xpdan/tests/utils.py

```diff
@@ -60,15 +60,16 @@ def insert_imgs(mds, fs, n, shape, save_dir=tempfile.mkdtemp()):
             descriptor=descriptor,
             uid=str(uuid4()),
             time=time.time(),
-            data={'img': fs_uid},
-            timestamps={},
+            data={'pe1_image': fs_uid},
+            timestamps={'pe1_image': time.time()},
             seq_num=i)
     mds.insert_run_stop(run_start=run_start,
                         uid=str(uuid4()),
                         time=time.time())
     imgs = [np.ones(shape)] * n
     run_start = mds.insert_run_start(uid=str(uuid4()), time=time.time(),
-                                     name='test', dark_uid=dark_uid)
+                                     name='test', dark_uid=dark_uid,
+                                     sc_dk_field_uid=dark_uid)
     data_keys = {
```

Comment:
if you wish to align with current xpdAcq md schema, probably just need ``sc_dk_field_uid``